src/check_sentryio_limits.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+import requests
+import sys
+
+
+p = ArgumentParser()
+p.add_argument('-b', '--bearer', type=str, required=True, dest='bearer',
+               help='A sentry api token with at least read permissions')
+p.add_argument('-o', '--organization', type=str, required=True,
+               dest='organization', help='The organization slug for the\
+               sentry.io organization to be queried')
+p.add_argument('-t', '--team', type=str, action='append', dest='teams',
+               help='Only check this team, can be added repeatetly')
+p.add_argument('-g', '--globallimt', type=int, dest='globallimit',
+               help='If the total amount if events per minute is higher \
+               than this limit the script will exit with a warning and \
+               a the exit code 1, for nrpe compatibility')
+p.add_argument('-p', '--perteamlimit', type=int, dest='perteamlimit',
+               help='If any teams\' projects\' keys summed up limits is \
+               higher than this, or not set the script will exit with a \
+               warning and the exit code 1')
+p.add_argument('-v', '--verbose', action='store_true', dest='verbose',
+               help='Print detailed stats')
+args = p.parse_args()
+
+
+def get_teams(organization, bearer):
+    '''Return a list of all teams in the account'''
+    teams = []
+    headers = {'Authorization': 'Bearer  {}'.format(bearer)}
```

Comment:
```suggestion
    headers = {'Authorization': f'Bearer  {bearer}'}
```
With Python 3.7 we can use the more readable f-string format :-)

Reply:
changed